service-front/assets/js/moj/moj.modules/moj.polyfill.js

```diff
@@ -13,6 +13,33 @@
 
     var NATIVE_DETAILS = typeof document.createElement('details').open === 'boolean'
 
+    /**
+     * restore tabbing on a list of focusable elements.
+     * fixes accessibility issue.
+     * @param elements
+     */
+    function restoreTabbing(elements){
```

Comment:
If you wanted to make this smaller, you could change the name to toggle and pass in a bool that can set or remove inside.

Reply:
good point - was being explicit about it's purpose. will take a look.